### PR TITLE
Release 1.0

### PR DESCRIPTION
**Pull request by yaroslavNqualisystems**
This change is [Reviewable](https://reviewable.io/reviews/qualisystems/cloudshell-checkpoint/4)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -19,7 +33,9 @@
     packages=find_packages(),
     install_requires=required,
     tests_require=required_for_tests,
-    python_requires="~=2.7",
+    python_requires=(
+        ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, !=3.6.*, <4"
+    ),
```
Comment (saklar13) [inline setup.py]: we only support python 3.7 and up, right? you can set ~=3.7

File: test_requirements.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
-mock==2.0.0
-nose==1.3.7
-coverage
+mock; python_version <= '2.7'
```
Comment (saklar13) [inline test_requirements.txt]: for python3 we don't need it. remove

**Review by saklar13 — CHANGES_REQUESTED** [review]

**Review by saklar13 — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (yaroslavNqualisystems) [reply setup.py]: Fixed

File: test_requirements.txt (hunk 2 above)
Comment (yaroslavNqualisystems) [reply test_requirements.txt]: Fixed

**Pull request merged**